model/testresult/testresult.go

```diff
@@ -26,6 +26,21 @@ type TaskTestResultsStats struct {
 	FilteredCount *int `json:"filtered_count" bson:"-"`
 }
 
+// TestResultsInfo describes information unique to a single task execution.
+type TestResultsInfo struct {
+	Project         string `bson:"project"`
+	Version         string `bson:"version"`
+	Variant         string `bson:"variant"`
+	TaskName        string `bson:"task_name"`
+	DisplayTaskName string `bson:"display_task_name,omitempty"`
+	TaskID          string `bson:"task_id"`
+	DisplayTaskID   string `bson:"display_task_id,omitempty"`
+	Execution       int    `bson:"execution"`
+	RequestType     string `bson:"request_type"`
```

Comment:
Could we call this `Requester`? It's the standard terminology used in Evergreen.

Reply:
Sure, though keeping the bson tag as-is for backward compatability